security/azure/restrict_rdp_internet/README.md

```diff
@@ -0,0 +1,36 @@
+# Azure Network Security Groups with inbound 3389 open
+
+## What it does
+
+This policy checks all azure subscriptions for Network Security Groups that have 3389 open to the internet.
+
+## Functional Details
+
+The policy leverages the Azure API to identify the network security groups which have port 3389 or a port range containing 3389 open to the internet and produces a report on them.
```

Comment:
Network Security Groups should be capitalized (since it is the official name of a type of resource in Azure).

This section should be more specific on the technical details, such as "This policy connects to the Azure Resource Manager API to get a list of Network Security Groups. It then checks the rules field of each group to see if any rules open port 3389" or something to that effect.

Reply:
Changed to "This policy connects to the Azure Resource Manager API to get a list of Network Security Groups. It then checks the rules field of each group to see if any rules are open inbound on port 3389"